opm/simulators/wells/MultisegmentWellPrimaryVariables.cpp

```diff
@@ -403,7 +403,7 @@ copyToWellState(const MultisegmentWellGeneric<Scalar>& mswell,
     // Note: for the ALQ value, in the StandardWell, WellInterfaceGeneric::getALQ(well_state) is used.
     // We might want to unify the way regarding AQL value.
     WellBhpThpCalculator(well_)
-        .updateThp(rho, stop_or_zero_rate_target, [this]() { return well_.wellEcl().alq_value(); },
+        .updateThp(rho, stop_or_zero_rate_target, [this, summary_state]() { return well_.wellEcl().alq_value(summary_state); },
```

Comment:
Tiny nit: Can you make the capture list be `[this, &summary_state]`, please?  That way we don't create a copy of the `summary_state` object inside the lambda.

Reply:
Ah, of course, should have known better. Thanks!